src/sql/azdata.proposed.d.ts

```diff
@@ -2612,6 +2612,8 @@ declare module 'azdata' {
 	export class TreeItem extends vscode.TreeItem {
 		payload?: IConnectionProfile;
 		childProvider?: string;
+		isServer?: boolean;
```

Comment:
This change doesn't seem correct from initial glance.  TreeNodes can be more that just servers or databases, so seems strange to have special flags for these.  Might be worth walking through this PR during the design review hour this afternoon.